src/spgwu/simpleswitch/spgwu_s1u.cpp

```diff
@@ -101,7 +101,8 @@ void spgwu_s1u_task(void* args_p) {
 spgwu_s1u::spgwu_s1u()
     : gtpu_l4_stack(
           spgwu_cfg.s1_up.addr4, spgwu_cfg.s1_up.port,
-          spgwu_cfg.s1_up.thread_rd_sched_params, spgwu_cfg.gtp_ext_hdr) {
+          spgwu_cfg.s1_up.thread_rd_sched_params,
+          spgwu_cfg.upf_5g_features.enable_5g_features) {
```

Comment:
@orion-belt 

so pretty much you removed that parameter and just used the 5g-feature boolean to replace it?

Reply:
@rdefosse 
yes you are right.